studio/desk-structure.js

```diff
@@ -1,8 +1,20 @@
 import S from '@sanity/desk-tool/structure-builder';
-import { MdHome, MdLayers, MdSettings } from 'react-icons/md';
+import { MdBuild, MdHome, MdLayers, MdSettings } from 'react-icons/md';
 
 const hiddenDocTypes = (listItem) =>
-  !['homepage', 'siteSettings', 'page'].includes(listItem.getId());
+  ![
+    'homepage',
+    'siteSettings',
+    'page',
+    'aboutPage',
+    'contactPage',
+    'footer',
+    'indivPage',
+    'lentPage',
+    'osloJazzPage',
+    'servicesPage',
+    'tabetaltPage',
```

Comment:
I think it might be an improvement if we normalized these pages into one `page` schema/type. Sanity has some [do's and don'ts](https://www.sanity.io/docs/donts-and-dos#e3797cd9ae57) that you could check out.

Reply:
Do you mean you want to create one dynamic schema for all the (and possibly new) pages that will be composed of the selected Sanity objects/forms?

Reply:
Yes 👍

Reply:
@cobraz, do you want to keep all content in Sanity (even some text on each of the pages)?
Some data can be categorized into entities (like services, team members, clients and etc.), but each of the pages has distinctive features from the others, like some images or text that defy a common structure for everyone. Should such data be stored in Sanity too?